src/SFA.DAS.EmployerAccounts/Services/PensionRegulatorService.cs

```diff
@@ -41,9 +41,9 @@ public async Task<IEnumerable<Organisation>> GetOrgansiationsByPayeRef(string pa
 
         private string GetBaseUrl()
         {
-            var baseUrl = _configuration.PensionRegulator.BaseUrl.EndsWith("/")
-                ? _configuration.PensionRegulator.BaseUrl
-                : _configuration.PensionRegulator.BaseUrl + "/";
+            var baseUrl = _configuration.PensionRegulatorApi.BaseUrl.EndsWith("/")
+                ? _configuration.PensionRegulatorApi.BaseUrl
+                : _configuration.PensionRegulatorApi.BaseUrl + "/";
 
             return baseUrl;
         }

```

Comment:
This will need corresponding changes in das-employer-config

Reply:
Already committed